setup.cfg

```diff
@@ -31,7 +31,7 @@ install_requires =
     attrs >= 20.1.0
     nibabel >=3.0.1
     nipype >=1.5.1,<2.0
-    niworkflows >= 1.4.2, < 1.6
+    niworkflows >= 1.4.8, < 1.6, !=1.5.0, !=1.5.1, !=1.5.2, !=1.5.3
```

Comment:
@mgxd Any reason not to release a 1.5.4 right away?

Reply:
Nope :)